tsconfig.json

```diff
@@ -22,7 +22,7 @@
     "next-env.d.ts",
     "**/*.ts",
     "**/*.tsx"
-  ],
+, "src/components/modules/MainPage/ShortCard/StyledShortCard.js"  ],
```

Comment:
this should be removed, we don't want to miss with ts config

Reply:
removed